PlayCover/AppInstaller/Installer.swift

```diff
@@ -118,7 +131,7 @@ class Installer {
                 InstallVM.shared.next(.finish, 0.95, 1.0)
                 returnCompletion(finalURL)
             } catch {
-                Log.shared.error(error)
```

Comment:
It would be easier to read if the `returnError` function returned the localised string and this is kept to `Log.shared.error`.

Reply:
So `returnError(error) -> String` and `Log.shared.error(returned_value)` ?

Reply:
Yep.